packages/manager/modules/pci/src/projects/project/storages/containers/add/add.controller.js

```diff
@@ -190,15 +190,38 @@ export default class PciStoragesContainersAddController {
           CONTAINER_USER_ASSOCIATION_MODES.CREATE
             ? 'pci_projects_project_storages_containers_add_success_message_with_user_creation'
             : 'pci_projects_project_storages_containers_add_success_message';
-        return this.goBack(
-          this.$translate.instant(message, {
+        const containerType = this.container.containerType
+          ? `${this.container.containerType}::`
+          : '';
+
+        return this.goBackWithTrackingPage({
+          message: this.$translate.instant(message, {
             container: this.container.name,
             userName: this.userModel.createMode?.user?.username,
           }),
-        );
+          trackingTag: `_add::${
+            this.container.offer === 'storage'
+              ? 'standard'
+              : this.container.offer
+          }_${
+            this.container.region?.datacenterLocation
```

Comment:
`datacenterLocation` could be empty or absent ?
if yes, tracking team except that hit include an `undifined`.